V3.0/UFTP-V3.0-metering.xsd

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
+
+    <xs:include schemaLocation="UFTP-V3.0-common.xsd"/>
+    <xs:include schemaLocation="UFTP-V3.0-agr-dso.xsd"/>
+
+    <!-- -->
+    <!-- EAN Type -->
+    <!-- -->
+    <xs:simpleType name="EANType">
+        <xs:restriction base="xs:string">
+            <xs:pattern value="[Ee][0-9]{16}"/>
+        </xs:restriction>
+    </xs:simpleType>
+    <!-- -->
+    <!-- -->
+    <!--  Metering Unit Type -->
+    <!-- -->
+    <xs:simpleType name="MeteringUnitType">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="kW">
+                <xs:annotation>
+                    <xs:documentation>kW must be used with Power profile values.</xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="kWh">
+                <xs:annotation>
+                    <xs:documentation>kWh must be used with energy profile values (ImportEnergy,ExportEnergy,ImportMeterReading,ExportMeterReading).</xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+        </xs:restriction>
+    </xs:simpleType>
+    <!-- -->
+    <!--  Metering Profile Enum -->
+    <!-- -->
+    <xs:simpleType name="MeteringProfileEnum">
```

Comment:
Is MeteringProfileQuantity a better name? My english is not as good as yours, offcourse... It's a physical quantity..

Reply:
I didn't come up with the name profile, but see this as an enum of profiles.  If and when we come to merge the units with the quantities we should agree on a meaningful name for the combined quanity.  Leaving it as it is for now, because we didn't plan to change this in the change document and to preserve backwards compatibility.